### PR TITLE
Abram UI enhancements

### PR DESCRIPTION
**Pull request by Glacian22**
**Requirements**

- [ ] I have added test coverage for new or changed functionality
- [ ] I have followed the repository's [pull request submission guidelines](../blob/main/CONTRIBUTING.md#submitting-pull-requests)
- [ ] I have validated my changes against all supported platform versions

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

Provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

Provide a clear and concise description of any alternative solutions or features you've considered.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

File: bin/ldcli
Comment (mike-zorn) [inline bin/ldcli:1]: This shouldn't be here. Similarly with everything in `bin`.

File: package-lock.json
Comment (mike-zorn) [inline package-lock.json:1]: I don't think we want to change these files. The dev server build should remain isolated from the rest of the cli.

File: internal/dev_server/ui/src/App.tsx [hunk 1]
```diff
@@ -5,10 +5,13 @@ import {
   IconButton,
```
Comment (mike-zorn) [inline internal/dev_server/ui/src/App.tsx:5]: I'll leave the actual ui code review for @kwatkins-ld or @ld-kyee

**Review by ld-kyee — COMMENTED** [review]
Once you make the changes mike suggested, I'll review the FE code.

File: bin/ldcli
Comment (Glacian22) [reply bin/ldcli:1]: @mike-zorn do you mean the `bin` folder isn't supposed to be tracked in git? I didn't touch any of that, or modify the gitignore, how would you like me to resolve this? Sorry for being unfamiliar with the usual LD git practices.

Comment (mike-zorn) [reply bin/ldcli:1]: Hmmm, do you recall which command you ran to generate these? I don't have them in my local environment.

Comment (Glacian22) [reply bin/ldcli:1]: I wonder if I ran `npm i` at the root level instead of just the front end folder, I'll see what I can figure out

Comment (mike-zorn) [reply bin/ldcli:1]: ohhhhh. That would probably have done it.

**Comment by Glacian22** [comment]
Made @mike-zorn 's changes, think I'm ready for @ld-kyee to review FE code

**Review by kwatkins-ld — APPROVED** [review]

**Pull request merged**